### ISSUE TITLE
TFT txn naming convention using Stellar > BSC bridge

### ISSUE THREAD

**Issue by renauter** [threefoldfoundation/tft#149]
While using the Stellar > BSC bridge user get the following transaction information: 

![bridge_tx_qrcode](https://user-images.githubusercontent.com/73958772/165948420-f898b86c-2abb-4045-8c7f-eada1c392f79.png)

Even if there is a warning message user can get confused if he doesn’t scan the QR code to fill the transaction fields and decide to do it manually for some reason (e.g. using Desktop version of TFConnect)
Naming convention is different on TFConnect transaction form and on Bridge transaction information
“Message” field on the TFConnect transaction form corresponds to “Memo” on the Bridge transaction information and the correspondence is not obvious.

## Suggestions

Bridge transaction information should follow as much as possible the naming convention on TFConnect transaction form
Also it should be clear that both address and memo are required for the transaction to succeed 

- [ ] Change “Destination” text to “ Destination address” 
- [ ] Change “Memo” text to “Message / Memo”
- [ ] Reinforce warning message saying that “Message / Memo” field is mandatory
- [ ] Add a “copy to clipboard” option at the end of both mandatory fields (would help to let it even clearer)

source file:
https://github.com/threefoldfoundation/tft/blob/4c9900950723d165f23268dfb421e504979b6c64/bsc/bridges/stellar/frontend/components/Deposit.tsx

**Comment by renauter** [comment]
@sabrinasadik 
In order to have a consistent naming convention maybe it would also be a good thing to standardize on the TFConnect app side
On the TFT txn form changing the "Message" field to "Memo" or to "Message / Memo" would avoid some confusion

**Comment by sabrinasadik** [comment]
@ken-de-moor pls see request above and create ticket in right repo

**Comment by ken-de-moor** [comment]
> @sabrinasadik In order to have a consistent naming convention maybe it would also be a good thing to standardize on the TFConnect app side On the TFT txn form changing the "Message" field to "Memo" or to "Message / Memo" would avoid some confusion

I agree, the term used in stellar is Memo, not message, so we will change it on the TF connect side.

**Comment by sabrinasadik** [comment]
https://github.com/threefoldtech/wallet-next/issues/149
